### PR TITLE
SKYKit and SKYKitChat upgraded to 1.1.0

### PR DESCRIPTION
**Pull request by howawong**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ target 'Swift Chat Demo' do
   pod 'JSQMessagesViewController', '~> 7.3.4'
   pod 'SVProgressHUD', '~> 2.1.0'
 
-  pod 'SKYKit', '~> 0.21.0'
-  pod 'SKYKitChat', :git => 'https://github.com/SkygearIO/chat-SDK-iOS.git', :commit => '50fce84'
-  pod 'SKYKitChat/UI', :git => 'https://github.com/SkygearIO/chat-SDK-iOS.git', :commit => '50fce84'
+  pod 'SKYKit/Core', :git => 'https://github.com/SkygearIO/skygear-SDK-iOS.git', :tag => '1.1.0-alpha.0'
```
Comment (rickmak) [inline Podfile]: Please use release 1.1.0

**Review by rickmak — CHANGES_REQUESTED** [review]
The user list is empty. And the generated query is wrong. What I get from the server log is 

```
[
    "and",
    [
        "neq",
        null,
        null
    ],
    [
        "neq",
        {
            "$type": "keypath",
            "$val": "_id"
        },
        "6601e8b4-e173-4b33-92ad-dbcbc10ea279"
    ]
]
```

**Comment by rickmak** [comment]
The predicate is correct if the serach for name of user is not empty, and as follow:

```
    "predicate": [
        "and",
        [
            "ilike",
            {
                "$type": "keypath",
                "$val": "name"
            },
            "%rick%"
        ],
        [
            "neq",
            {
                "$type": "keypath",
                "$val": "_id"
            },
            "6601e8b4-e173-4b33-92ad-dbcbc10ea279"
        ]
    ],
```

**Comment by howawong** [comment]
query bug fix in skygeario/chat-SDK-iOS#51

**Pull request merged**